### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#1190] [General]
- General: 3 new icons
- General: Translation updates

<hr /><em>This discussion was created from the release <a href='https://github.com/stratumauth/app/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by ccoVeille** [comment]
How to import from old app?

**Reply by ccoVeille** [reply]
@jamie-mh maybe?

**Reply by Akip25** [reply]
instead of import (which the app prompts you to do on startup but doesn't give an option for authpro)

click the + in the bottom right and do "restore"

**Reply by ccoVeille** [reply]
Thanks
